recipes/gradio/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
The license is already included in the source tarball so you don't need to (and shouldn't) include it in the recipe. Please go ahead and remove.

Reply:
Done